workload/docs/citrixlzaccelerator/readme.md

```diff
@@ -363,7 +363,7 @@ Here is the JSON description of the Citrix Host Role with the minimum permission
 }	
 ```
 	
-The Citrix_Hosting_Connection custom role should be assigned to the Citrix_Infrastructure resource group(s) that have the Cloud Connector, Master Image or Virtual Network resources in them. This JSON role can be copied and pasted directly into your custom Azure AD role.
+The Citrix_Hosting_Connection custom role should be assigned to the Citrix_Infrastructure resource group(s) that have the Cloud Connector, Master Image or Virtual Network resources in them. This JSON role can be copied and pasted directly into your custom Microsoft Entra ID role.
```

Comment:
When talking about roles, it's always "Microsoft Entra role" -- why, IDK.  Just is :)

```suggestion
The Citrix_Hosting_Connection custom role should be assigned to the Citrix_Infrastructure resource group(s) that have the Cloud Connector, Master Image or Virtual Network resources in them. This JSON role can be copied and pasted directly into your custom Microsoft Entra role.
```